libkodimote/songs.cpp

```diff
@@ -210,10 +210,7 @@ void Songs::playItem(int row)
     } else {
         pItem.setAlbumId(m_albumId);
     }
-    Kodi::instance()->audioPlayer()->playlist()->clear();
-    Kodi::instance()->audioPlayer()->playlist()->addItems(pItem);
-//    Kodi::instance()->audioPlayer()->playlist()->playItem(row);
-    Kodi::instance()->audioPlayer()->playItem(row);
+    Kodi::instance()->audioPlayer()->open(pItem, resume);
```

Comment:
I think this is the correct place for the previous comment.

Is there a reason why you use open() for music too? IIRC we wanted to do that only for Videos where the resume actually makes sense and leave the Music part as is.